src/elements/content-sidebar/Sidebar.js

```diff
@@ -91,14 +96,68 @@ class Sidebar extends React.Component<Props, State> {
 
         this.state = {
             isDirty: this.getLocationState('open') || false,
+            isDocGenTemplate: false,
         };
 
         this.setForcedByLocation();
     }
 
+    disableDocgenTab() {
+        const { history, location } = this.props;
+        this.setState({
+            ...this.state,
+            isDocGenTemplate: false,
+        });
+        if (location.pathname === `/${SIDEBAR_VIEW_DOCGEN}`) {
+            history.push('/');
+        }
+    }
+
+    fetchMetadataSuccessCallback = ({ editors }: { editors: Array<MetadataEditor> }) => {
+        const { history } = this.props;
+        if (editors && editors.length > 0) {
+            const template = editors[0]?.template;
+            const scopeIsGlobal = template.scope === 'global';
```

Comment:
If editors[0] is null or undefined won't template by undefined? If that's the case then unless you do template?.scope below you'll get an error right?

Reply:
removed from this PR